libs/sdk-backend-tiger/src/backend/organization/settings.ts

```diff
@@ -26,7 +34,7 @@ export class OrganizationSettingsService
         return data.data.reduce((result: ISettings, setting) => {
             return {
                 ...result,
-                [setting.id]: unwrapSettingContent(setting.attributes?.content),
+                [setting.id]: unwrapSettingContent(setting.attributes),
```

Comment:
Just a confirmation: do we really need to make this change? Is it because the function `getSettings` does not work as expected?

Reply:
Because some settings has Dynamic ID (ex: Whitelabeling), so we should include the type of setting to result of function. After that, we can filter it on UI.
<img width="731" alt="Screen Shot 2023-04-03 at 16 08 56" src="https://user-images.githubusercontent.com/5329091/229735287-1fc22be2-bf61-4a4c-ac12-7e36fe4dab1c.png">

Reply:
I have updated new code at here. Please help me review again